source/MRCommonPlugins/ViewerButtons/MRIOFilesMenuItems.h

```diff
@@ -17,7 +17,6 @@ class OpenFilesMenuItem : public RibbonMenuItem, public MultiListener<DragDropLi
 private:
     virtual bool dragDrop_( const std::vector<std::filesystem::path>& paths ) override;
     void setupListUpdate_();
-    void loadFiles_( const std::vector<std::filesystem::path>& paths );
```

Comment:
why is this deleted?

Reply:
method don't used and haven't realization